tests/v2/validation/resource/terraform/v1charts/aws/main.tf

```diff
@@ -0,0 +1,233 @@
+  answers = {
+    "certmanager.enabled" = false
+    "enableCRDs" = true
+    "galley.enabled" = true
+    "gateways.enabled" = false
+    "gateways.istio-ingressgateway.resources.limits.cpu" = "2000m"
+    "gateways.istio-ingressgateway.resources.limits.memory" = "1024Mi"
+    "gateways.istio-ingressgateway.resources.requests.cpu" = "100m"
+    "gateways.istio-ingressgateway.resources.requests.memory" = "128Mi"
+    "gateways.istio-ingressgateway.type" = "NodePort"
+    "global.monitoring.type" = "cluster-monitoring"
+    "global.rancher.clusterId" = rancher2_cluster_sync.v1charts-sync.cluster_id
+    "istio_cni.enabled" = false
+    "istiocoredns.enabled" = false
+    "kiali.enabled" = true
+    "mixer.enabled" = true
+    "mixer.policy.enabled" = true
+    "mixer.policy.resources.limits.cpu" = "4800m"
+    "mixer.policy.resources.limits.memory" = "4096Mi"
+    "mixer.policy.resources.requests.cpu" = "1000m"
+    "mixer.policy.resources.requests.memory" = "1024Mi"
+    "mixer.telemetry.resources.limits.cpu" = "4800m",
+    "mixer.telemetry.resources.limits.memory" = "4096Mi"
+    "mixer.telemetry.resources.requests.cpu" = "1000m"
+    "mixer.telemetry.resources.requests.memory" = "1024Mi"
+    "mtls.enabled" = false
+    "nodeagent.enabled" = false
+    "pilot.enabled" = true
+    "pilot.resources.limits.cpu" = "1000m"
+    "pilot.resources.limits.memory" = "4096Mi"
+    "pilot.resources.requests.cpu" = "500m"
+    "pilot.resources.requests.memory" = "2048Mi"
+    "pilot.traceSampling" = "1"
+    "security.enabled" = true
+    "sidecarInjectorWebhook.enabled" = true
+    "tracing.enabled" = true
+    "tracing.jaeger.resources.limits.cpu" = "500m"
+    "tracing.jaeger.resources.limits.memory" = "1024Mi"
+    "tracing.jaeger.resources.requests.cpu" = "100m"
+    "tracing.jaeger.resources.requests.memory" = "100Mi"
+  }
+}
+
+resource "rancher2_cluster_logging" "cluster-logging" {
+  name = "cluster-logging"
+  cluster_id = rancher2_cluster.custom.id
+  kind = "syslog"
+  syslog_config {
+    endpoint = var.logging_endpoint
+    protocol = "udp"
+    severity = "notice"
+    ssl_verify = false
+  }
+}
+
+resource "rancher2_namespace" "longhorn-system-namespace" {
+  name = "longhorn-system"
+  project_id = rancher2_cluster_sync.v1charts-sync.system_project_id
+  description = "longhorn-system namespace"
+}
+
+resource "null_resource" "longhorn-iscsi-nfs" {
+  depends_on = [rancher2_cluster_sync.v1charts-sync,local_file.sync_kube_config,rancher2_namespace.longhorn-system-namespace]
+  provisioner "local-exec" {
+    environment = {
+      KUBECONFIG = "kube_config_provisioned_cluster.yaml"
+    }
+    command = "kubectl apply -f https://raw.githubusercontent.com/longhorn/longhorn/v1.3.0/deploy/prerequisite/longhorn-iscsi-installation.yaml --namespace=longhorn-system && kubectl apply -f https://raw.githubusercontent.com/longhorn/longhorn/v1.3.0/deploy/prerequisite/longhorn-nfs-installation.yaml --namespace=longhorn-system"
```

Comment:
Should this be hardcoded? Will we ever want to change the version?

Reply:
Added Longhorn prereq variable and set current defaults for 2.5 and 2.6
	- This removes the hardcoded version number